Sources/CorePermissionsSwiftUI/Modifiers/Public/MainTextModifiers.swift

```diff
@@ -18,7 +18,7 @@ public extension CustomizableView {
      - Parameter _: The custom text to change to
      */
     
-    @inlinable func changeHeaderTo(_ text:String) -> some CustomizableView {
+    @inlinable func changeHeaderTo(_ text: String) -> some CustomizableView {
```

Comment:
The formatting changes are fine, just in the future, do them in a separate PR so it's easier for me to see the important logic code stuff.